mwim.el

```diff
@@ -124,38 +124,63 @@ to the end of line."
           (goto-char eoc)))))
   (skip-chars-backward " \t"))
 
+(defun mwim-forward-line (arg)
+  "Move ARG - 1 lines forward (backward if ARG is negative).
+If ARG is nil or 1 do nothing."
+  (interactive "^p")
+  (setq arg (or arg 1))
+  (when (/= arg 1)
+    (let ((line-move-visual nil))
+      (forward-line (1- arg)))))
+
```

Comment:
After all, I would write this function like this:

```elisp
(defun mwim-forward-line (&optional arg)
  "Move ARG - 1 lines forward (backward if ARG is negative).
If ARG is nil or 1 do nothing."
  (when (and arg (/= arg 1))
    (forward-line (1- arg))))
```

But I think this is a strange function: it is used to "forward line",
but not when ARG is 1 (why?), and when you need to go to the next line
you need to give ARG=2. :confused: